src/showItems.js

```diff
@@ -0,0 +1,99 @@
+import { updateStatus } from './updateStatus.js';
+import {
+  listAddTask, updateDescription, listUpdateIndexes, listRemoveTask,
+} from './listManager.js';
+
+export const setTaskListeners = (tasksList, index) => {
+  const text = document.querySelector(`.item-${index} textarea`);
+
+  text.addEventListener('change', () => {
+    updateDescription(tasksList, index, text);
+  });
+
+  text.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      updateDescription(tasksList, index, text);
+      document.activeElement.blur();
+    }
+  });
+
+  text.addEventListener('focus', () => {
+    const icon = text.nextElementSibling;
+    icon.classList.remove('fa-ellipsis-v');
+    icon.classList.add('fa-trash-alt');
+  });
+
+  text.addEventListener('blur', () => {
+    const icon = text.nextElementSibling;
+    icon.classList.add('fa-ellipsis-v');
+    icon.classList.remove('fa-trash-alt');
+  });
+
+  const check = document.querySelector(`.item-${index} input`);
+  check.addEventListener('change', () => { updateStatus(tasksList, index); });
+
+  const rmBtn = document.querySelector(`.item-${index} i`);
+  // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Please remove this eslint disable line, it's best to fix the linting issue in the code.